cunumeric/module.py

```diff
@@ -2974,13 +3092,17 @@ def tensordot(a, b, axes=2, out=None):
 
 
 # Trivial multi-tensor contraction strategy: contract in input order
-class NullOptimizer(oe.paths.PathOptimizer):
-    def __call__(self, inputs, output, size_dict, memory_limit=None):
+class NullOptimizer(oe.paths.PathOptimizer):  # type: ignore
+    def __call__(  # type: ignore [no-untyped-def]
+        self, inputs, output, size_dict, memory_limit=None
+    ) -> list[tuple[int, int]]:
```

Comment:
Once things are installed to a regular env site-packages, these ignores might be able to go away